doajtest/unit/test_task_journal_bulkedit.py

```diff
@@ -167,7 +169,7 @@ def test_05_edit_metadata(self):
                                  doaj_seal=True,
                                  country="AF",
                                  owner="test1",
-                                 platform="my platform",
+                                 platform=u"my platfo®m",   # stick in a weird character for good measure
```

Comment:
This should be okay here .. in general characters from other alphabets will provide a higher chance of breaking unicode decoding and encoding.

```
>>> u"®".encode('ascii')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeEncodeError: 'ascii' codec can't encode character u'\xae' in position 0: ordinal not in range(128)
>>> u"®".encode('latin-1')
'\xae'  # <-------- it worked with latin-1, a common web browser default
>>> u"асд".encode('latin-1')
Traceback (most recent call last):  # <------------ no such luck with Cyrillic
  File "<stdin>", line 1, in <module>
UnicodeEncodeError: 'latin-1' codec can't encode characters in position 0-2: ordinal not in range(256)
```

Googling "some cyrillic" and "some chinese" yields quick results. If in doubt, you can always refer to http://unicodesnowmanforyou.com/ , which I've only managed to encode in utf-8 from the common codecs.